src/lsm/tree.zig

```diff
@@ -2593,7 +2725,7 @@ pub fn main() !void {
     _ = tree.get;
     _ = tree.put;
     _ = tree.remove;
-    _ = tree.lookup;
+    _ = tree.lookup_value;
```

Comment:
Let's keep this as `lookup()` and then rather use `context` or something like that instead of `lookup` as the local variable within the `Lookup` struct.

Reply:
Fine by me, my thought was to mirror `Grid.read_block()` and `Grid.Read` but I also agree that `lookup()` is the nicer name to expose outside of the tree.

Reply:
renamed `lookup_value()` -> `lookup()` and `lookup: *Lookup` to `context: *LookupContext`.